blocks/flip-cards/flip-cards.js

```diff
@@ -1,16 +1,67 @@
-export default function decorate(block) {
+async function getHtml(url) {
```

Comment:
Not sure, but just some food for thought: Wouldn't it be more performant if we just fetch the title and description from the index instead of fetching the whole DOM of the pages?

Reply:
Done! Thanks for the suggestion!